docs/Concepts/Merge.md

```diff
@@ -22,29 +25,36 @@ Execution and consensus clients communicate with each other using the
 
 ### Execution clients
 
-Execution clients, such as [Besu], manage the state and execute transactions on the execution layer.
+Execution clients, such as [Besu](https://besu.hyperledger.org/en/stable/), manage the state and execute transactions on
+the execution layer.
 Execution clients serve [JSON-RPC API](https://besu.hyperledger.org/en/stable/Reference/API-Methods/) requests and
 communicate with each other in a peer-to-peer network.
 
 ### Consensus clients
 
 Consensus clients, such as Teku, contain beacon node and validator client implementations.
-The beacon node is the primary link to the [Beacon Chain] (consensus layer).
+The beacon node is the primary link to the [Beacon Chain](https://ethereum.org/en/upgrades/beacon-chain/) (consensus layer).
 The validator client performs [validator duties](Proof-of-Stake.md) on the consensus layer.
 Consensus clients serve [REST API](../Reference/Rest_API/Rest.md) requests and communicate with each other in a
 peer-to-peer network.
 
-### Run a node
+## What happens during The Merge
+
+Before The Merge, the consensus client's configuration will be [updated](../HowTo/Prepare-for-The-Merge.md#update-teku)
+to listen for a certain total terminal difficulty (TTD) to be reached on the
+[execution endpoint](../Reference/CLI/CLI-Syntax.md#ee-endpoint).
```

Comment:
Both the consensus client and execution client will be updated to know the TTD in preparation for the merge. Teku will periodically request information about the execution client's configuration to check it can connect and they agree on the config. It will log warnings if the configuration doesn't match.